spec/features/groceries_spec.rb

```diff
@@ -9,16 +9,21 @@
 
       let(:new_item_name) { 'blueberries' }
 
-      it 'allows adding an item' do
+      it 'allows adding an item, deleting an item, undoing the deletion, and checking in a shopping trip' do
```

Comment:
(The "checking in a shopping trip" specs had been added previously, but they weren't added to this example description, which they should have been. Doing so now.)